en/admin/admin_utils.rst

```diff
@@ -666,9 +666,9 @@ The following shows [options] available with the **cubrid copydb** utility.
 
         cubrid copydb -d -F /home/usr/CUBRID/databases demodb new_demodb
 
-.. option:: --copy-lob-path=PATH
+.. option:: --copy-lob-path
 
-    If the **\-\-copy-lob-path** option is specified, a new directory path for LOB files is created and a source database is copied into a new directory path. If this option is omitted, the directory path is not created. Therefore, the **lob-base-path** of the **databases.txt** file should be modified separately. This option cannot be used with the **-B** option. ::
+    If the **\-\-copy-lob-path** option is specified, lob directory path of the target database will be set to the lob directory path of the source database. If this option is omitted, lob directory path of the target database will be set to <target database directory path>/lob. This option does not copy lob files and cannot be used with the **-B** option. ::
```

Comment:
```suggestion
    If the **\-\-copy-lob-path** option is specified, the lob directory path of the target database will be set to the lob directory path of the source database. If this option is omitted, the lob directory path of the target database will be set to <target database directory path>/lob. This option does not copy lob files and cannot be used with the **-B** option. ::
```

Reply:
@junsklee , thank your for your comment!